### DISCUSSION TITLE
Some words for those who are causing disturbance within the issue.|一些对在Issue里制造骚扰的人说的话

### DISCUSSION THREAD

**Discussion by 443eb9** [xai-org/grok-1#245] [General]
实在忍不住了来说两句
开源大模型本来就不是什么容易的事情，还被一堆人发issue骚扰，真的是看得我反胃
有些人的素质有待提高，没人逼你说话，不会说，不知道在哪里说你可以不说
发一些莫名其妙的东西觉得自己很幽默吗？笑死了

I can't help but say a few words.
Working on an open-source large model is not an easy task, and being harassed by a bunch of people issuing issues really makes me sick.
Some people's quality still needs to be improved. No one is forcing you to speak. If you don’t know where to speak or don’t know how to speak, you can just stay quiet.
Do you think you're being funny by posting some inexplicable things? It's just ridiculous.

**Comment by 443eb9** [comment]
还要专门发issue来提醒某些没教养的真的是抽象
我tm替你们感到丢脸！ :nauseated_face:

**Reply by ghost** [reply]
毕竟大学高中加起来才30%，你指望他们能理解Issues和Discussions就有点不切实际了😋

**Comment by nwq115212** [comment]
你也别把自己当多高尚
你骂的话我还是很赞同的

**Reply by 443eb9** [reply]
没把自己当成怎么样的人，哥们没那个资格
单纯看到某些人干的事情感到很生气

**Comment by nwq115212** [comment]
Don't regard yourself as exceedingly noble, yet I find myself in agreement with the criticisms you've articulated.
